### DISCUSSION TITLE
Fixing syntax highlighting in VSCode

### DISCUSSION THREAD

**Discussion by dflvunoooooo** [leo-colisson/robust-externalize#29] [General]
I asked in the VSCode LATEX-Workshop repository to fix the syntax highlighting for the robust-extrernalize environments. But it is of cause rather complicated. Would you @tobiasBora be so kind to take a look [here (LaTeX-Workshop/issues/4145)](https://github.com/James-Yu/LaTeX-Workshop/issues/4145) and give an overview, which environments are there to be incorporated. If you have the time. :)

**Comment by dflvunoooooo** [comment]
Syntax highlighting is advancing. But there are some questions about options for environments. @tobiasBora would you be so kind to clarify this at [Latex Workshop PR79](https://github.com/jlelong/vscode-latex-basics/pull/79)?

**Reply by tobiasBora** [reply]
Yeah, they are doing a pretty good job.

**Reply by dflvunoooooo** [reply]
Thank you for your support.
